core/single-file/README.md

```
@@ -1,4 +1,8 @@
-# @stephansama/single-file
+<div align="center">
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)